src/praisonai-train/praisonai_train/train/llm/trainer.py

```
@@ -149,6 +158,26 @@ def __init__(self, config_path="config.yaml"):
+        config dict, skipping the training-only validation (no dataset required).
+
+        Used by the standalone `export` command so a non-developer can publish a
+        model that was trained earlier without re-running training. The caller loads
+        the model via load_model() and assigns self.model / self.hf_tokenizer.
+        """
+        obj = cls.__new__(cls)
+        _lazy_import_training_deps()
+        obj.config = dict(config or {})
+        if "model" in obj.config and "model_name" not in obj.config:
+            obj.config["model_name"] = obj.config["model"]
+        obj.device = torch.device("cuda" if torch.cuda.is_available() else "cpu")
```

Ruff (0.15.21)

[error] Undefined name `torch`

(F821)